### PR TITLE
i18n(ko-KR): update `typescript.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `typescript.mdx`
- #9240 
- #10148
- #10197

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by jsparkdev** [comment]
Once #10166 is merged, I expect to be able to merge this PR successfully.

**Review by yanthomasdev — APPROVED** [review]
LGTM! I'll merge the umbrella PR once I approve all of the individual PRs.

**Pull request closed without merging**
